rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/10_basic.yml

```diff
@@ -50,12 +50,12 @@ setup:
       indices.get_data_stream:
         name: "*"
   - match: { 0.name: simple-data-stream1 }
-  - match: { 0.timestamp_field: '@timestamp' }
+  - match: { 0.timestamp_field.field_name: '@timestamp' }
```

Comment:
I wonder if this should be just `timestamp_field.name`?

I would like to also see an assert on the mapping being included and having the right content.